static/app/views/organizationGroupDetails/allEventsTable.tsx

```diff
@@ -14,7 +16,8 @@ export interface Props {
 
 const AllEventsTable = (props: Props) => {
   const {location, organization, issueId, isPerfIssue} = props;
-  const eventView: EventView = EventView.fromLocation(location);
+  const [error, setError] = useState<string | undefined>(undefined);
```

Comment:
I agree, and a empty string is treated as falsey so that will work for my code, nice catch.
This does assume that the error message will never actually be an empty string, which seems like a good assumption to me.
Will update it